docs/docusaurus.config.js

```diff
@@ -34,10 +37,6 @@ const codeTheme = require('./src/utils/prism');
             showLastUpdateAuthor: true,
             showLastUpdateTime: true,
           },
-          gtag: {
-            trackingID: 'G-ZC17SLMXRN',
-            anonymizeIP: true,
-          },
```

Comment:
Not sure why those changes are here, did you merged from "next" ?

Reply:
earlier I have merged from main but later on merged it from next